File: KuantumKaosYonetimi/Python/KuantumKaosYonetimi/veri_paketi.py
# veri_paketi.py
from .kuantum_nesnesi import KuantumNesnesi

# C. Nesne Çeşitleri: VeriPaketi (IKritik değil)
class VeriPaketi(KuantumNesnesi):
    
    def __init__(self, id_str: str, stabilite: float):
        # KuantumNesnesi'nin yapıcı metodunu çağırır. Tehlike seviyesi: 1
        super().__init__(id_str, 1, stabilite) 

    # AnalizEt() (Çok Biçimlilik/Polymorphism)
    def AnalizEt(self):
        # Stabilite sadece 5 birim düşer.
        self.Stabilite -= 5
        print(f"[ID: {self.ID}] Veri içeriği okundu. Stabilite -5. Yeni Stabilite: {self.Stabilite:.2f}%")

File: KuantumKaosYonetimi/Java/KuantumKaosYonetimi/VeriPaketi.java
// VeriPaketi.java
package KuantumKaosYonetimi;

public class VeriPaketi extends KuantumNesnesi {
    
    public VeriPaketi(String id, double stabilite) {
        // Tehlike Seviyesi: 1 (Düşük)
        super(id, 1, stabilite); 
    }

    // AnalizEt() (Çok Biçimlilik/Polymorphism)
    @Override
    public void AnalizEt() {
        // Stabiliteyi 5 birim düşür
        setStabilite(getStabilite() - 5);
        System.out.printf("[ID: %s] Veri içeriği okundu. Stabilite -5. Yeni Stabilite: %.2f%%%n", 
                          getID(), getStabilite());
    }
}